docs/manual/manual_underactuated_torque_correction.rst

```diff
@@ -0,0 +1,51 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _correcting_underactuated_torque:
+
+.. |propeller_name| replace:: UP12
+.. |motor_name| replace:: Vertiq 2306
+
+***********************************************
+Underactuated Propeller Torque Angle Correction
+***********************************************
+
+Underactuated propellers can vector force, torque, or a combination of the two. The flapping blades on the |propeller_name| vectors both a force and torque vector. These two vectors are coupled, and always point the same direction relative to each other. In the case of the |propeller_name|, they are approximately 82° apart. This means that the torque vector and torque induced by the force vector happen at slightly different angles. Depending on the height of the propeller above the Center of Mass (CoM), the resulting torque will point in different directions. Some plots showing different heights above the center of mass show how the final torque direction and magnitude can change.
+
+.. figure:: ../_static/manual_images/pulsing_propeller/torque_angle_0.png
+    :align: center
+    :scale: 50
+    :alt: CCW |propeller_name| 0mm above CoM
+
+    CCW |propeller_name| 0mm above CoM
```

Comment:
Are Direct Torque and Torque Generated By Force pointing in the same direction as total torque here? I can't see them. May be good to clarify where they are in the caption.